docs/reference/glossary.rst

```diff
@@ -127,6 +127,13 @@ Glossary
       RBNO is enabled by default for a subset node operations. 
       See :doc:`Repair Based Node Operations </operating-scylla/procedures/cluster-management/repair-based-node-operation>` for details.
 
+    RF-rack-valid keyspace
+      A keyspace with :doc:`tablets </architecture/tablets>` enabled is RF-rack-valid if all of its data centers
+      have the :term:`Replication Factor (RF) <Replication Factor (RF)>` of 0, 1, or the number of racks with at least
```

Comment:
We can mention Virtual Racks https://github.com/scylladb/scylladb/issues/23260 here 
(can be a followup PR)